app/stac_api/validators.py

```diff
@@ -187,3 +187,29 @@ def validate_item_properties_datetimes(
             properties_start_datetime,
             properties_end_datetime,
         )
+
+
+def validate_JSON_payload_write_requests(serializer_class, request_data):  # pylint: disable=invalid-name
+    '''
+    Validate JSON payload of write requests.
+    Args:
+         serializer_class: serializer class for validating and deserializing input and
+         for serializing output.
+
+         request_data: data of the request to be validated
+    Raises:
+        ValidateionError: Either in case of unexpected payload or if the payload contains
+        values for read-only fields.
+    '''
+    expected_payload = list(serializer_class().fields.keys())
```

Comment:
only create one serializer instance, but for the future asset management it would be better to already have the view instance here.

Reply:
I changed it so that now the serializer instance is passed to this function. Did not really get the point with the view class so far. For what is it needed here?